libs/common/src/helpers/is-feature-flag-enabled.test.ts

```diff
@@ -44,6 +49,43 @@ describe('is-feature-flag-enabled helpers', () => {
     });
   });
 
+  describe('isTfmFacilityEndDateFeatureFlagEnabled', () => {
+    const tfmFacilityEndDateFeatureFlag: FeatureFlag = 'FF_TFM_FACILITY_END_DATE_ENABLED';
+
+    it("returns false when the feature flag is set to 'false'", () => {
```

Comment:
could we extract these tests out into a test helper (ie search for `with-validate...` and youll get some examples. Maybe something like 

```
withBooleanFeatureFlagTests({
    featureFlagName: tfmFacilityEndDateFeatureFlag
    getFeatureFlagValue: isTfmFacilityEndDateFeatureFlagEnabled
})
```

I'd even be happy to extract the `describe` block here so that this test file has basically 3 lines of `withBooleanFeatureFlagTests`, one for each of the env vars :) 

I know this is a bit of a refactor, but I don't really want this test file to get much longer!

What do you think?

Reply:
A great idea, saves a lot of duplication. I've given that a go, see what you think! 

I'm actually unsure how to run the tests in this`is-feature-flag-enabled.test.ts` file - what command should I run here / which directory do I run it from? 👀